django_rq/workers.py

```diff
@@ -1,25 +1,40 @@
 from rq import Worker
 from rq.utils import import_attribute
 
-from .queues import get_queues
 from .settings import EXCEPTION_HANDLERS
+from .queues import get_queues
 
 
-def get_exception_handlers():
-    """
-    Custom exception handlers could be defined in settings.py:
-    RQ = {
-        'EXCEPTION_HANDLERS': ['path.to.handler'],
+def get_exception_handlers(exception_handlers):
```

Comment:
I think it's better to change the function signature to `get_exception_handlers(queue_name)`